src/jit/gentree.cpp

```diff
@@ -4358,25 +4358,25 @@ unsigned Compiler::gtSetEvalOrder(GenTree* tree)
 
         case GT_ARR_ELEM:
 
-            level  = gtSetEvalOrder(tree->gtArrElem.gtArrObj);
-            costEx = tree->gtArrElem.gtArrObj->GetCostEx();
-            costSz = tree->gtArrElem.gtArrObj->GetCostSz();
+            level  = gtSetEvalOrder(tree->AsArrElem()->gtArrObj);
```

Comment:
Please add a local var (and a new `{ }`) here.

Reply:
I was not able to do these changes. They are causing compilation failures.

Reply:
That is strange, I have committed these changes to your PR, lets see if ci shows any failures.

Reply:
I do not see any failures.

Reply:
strange, I rebased the branch. It is working for me too. I must have some kind of parenthesis forgotten somewhere.